database/scripts/0001-create-initial-tables.sql

```diff
@@ -51,4 +51,24 @@ CREATE TABLE public.event_rating
     event_id INTEGER NOT NULL REFERENCES public.events (id),
 
     CONSTRAINT unique_user_per_event UNIQUE (created_by, event_id)
-);
\ No newline at end of file
+);
+
+CREATE TABLE public.user_settings
+(
+    id SERIAL PRIMARY KEY,
+    receive_top_rated BOOLEAN NOT NULL DEFAULT false,
```

Comment:
Maybe `receive_top_rated_notification` for clarity?

Reply:
receive_top_rated_notification**s**?
I think it better describes "Receive notifications about top-rated sport events", if that was the intention of this setting to begin with

Reply:
Looking good!